src/main/java/com/minecolonies/coremod/tileentities/ScarecrowTileEntity.java

```diff
@@ -1,10 +1,11 @@
 package com.minecolonies.coremod.tileentities;
 
 import com.minecolonies.api.util.EntityUtils;
+import com.minecolonies.api.util.LanguageHandler;
+import com.minecolonies.api.util.constant.Constants;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.constant.Constants'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)